kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/io/kotest/matchers/collections/DescribePartialMatchesInCollectionTest.kt

```diff
@@ -0,0 +1,52 @@
+package io.kotest.matchers.collections
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContainInOrder
+
+class DescribePartialMatchesInCollectionTest: WordSpec() {
```

Comment:
Very nitpick: Why not use the constructor instead of init?

Reply:
just personal style - I'll try to use the constructor next time